src/common/models/essence/essence.mocha.ts

```diff
@@ -1,12 +1,91 @@
 'use strict';
 
 import { expect } from 'chai';
+import { testImmutableClass } from 'immutable-class/build/tester';
+import { List } from 'immutable';
 
 import { $, Expression } from 'plywood';
 import { Essence } from './essence';
+import { DataSource } from "../data-source/data-source";
 
 describe('Essence', () => {
-  it('works in empty case', () => {
+  var isAutomatic = function(): boolean { return false; };
+  var testFunction = function(): any { return {'isAutomatic' : isAutomatic}; };
```

Comment:
Mt preference is to inline dummy stub functions like this just to make things a little more self contained.